Solutions/CyberArkEPM/data/Solution_CyberArkEPM.json

```diff
@@ -3,13 +3,42 @@
   "Author": "CyberArk Business Development - business_development@cyberark.com",
   "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/CyberArk_Logo.svg\" width=\"75px\" height=\"75px\">",
   "Description": "Endpoint Privilege Manager, a critical and foundational endpoint control addresses the underlying weaknesses of endpoint defenses against a privileged attacker and helps enterprises defend against these attacks.",
-  "Parsers": [
-	"Parsers/CyberArkEPM.txt"
-	],
-  "Data Connectors": [
-	"DataConnectors/CyberArkEPM_API_FunctionApp.json"
-	],
-  "Metadata": "SolutionMetadata.json",
-  "BasePath": "C:\\GitHub\\azure\\Solutions\\CyberArkEPM",
-  "Version": "1.0.0"
+"Data Connectors": [
+	"Solutions/CyberArkEPM/DataConnectors/CyberArkEPM_API_FunctionApp.json"
+],
+"Parsers": [
+	"Solutions/CyberArkEPM/Parsers/CyberArkEPM.txt"
+],
+ "Hunting Queries": [
+    "Solutions/CyberArkEPM/Hunting Queries/CyberArkEPMElevationRequests.yaml",
+    "Solutions/CyberArkEPM/Hunting Queries/CyberArkEPMPowershellDownloads.yaml",
+    "Solutions/CyberArkEPM/Hunting Queries/CyberArkEPMPowershellExecutionParameters.yaml",
+    "Solutions/CyberArkEPM/Hunting Queries/CyberArkEPMProcessNewHash.yaml",
+    "Solutions/CyberArkEPM/Hunting Queries/CyberArkEPMProcessesAccessedInternet.yaml",
+    "Solutions/CyberArkEPM/Hunting Queries/CyberArkEPMProcessesRunAsAdmin.yaml",
+    "Solutions/CyberArkEPM/Hunting Queries/CyberArkEPMRareProcVendors.yaml",
+    "Solutions/CyberArkEPM/Hunting Queries/CyberArkEPMRareProcessesRunByUsers.yaml",
+    "Solutions/CyberArkEPM/Hunting Queries/CyberArkEPMScriptsExecuted.yaml",                           
+    "Solutions/CyberArkEPM/Hunting Queries/CyberArkEPMSuspiciousActivityAttempts.yaml"
+],
+"Analytic Rules": [
+	"Solutions/CyberArkEPM/Analytic Rules/CyberArkEPMAttackAttemptNotBlocked.yaml",
+	"Solutions/CyberArkEPM/Analytic Rules/CyberArkEPMMSBuildLOLBin.yaml",
+	"Solutions/CyberArkEPM/Analytic Rules/CyberArkEPMMultipleAttackAttempts.yaml",
+	"Solutions/CyberArkEPM/Analytic Rules/CyberArkEPMNewProcessStartetFromSystem.yaml",
+	"Solutions/CyberArkEPM/Analytic Rules/CyberArkEPMPossibleExecutionOfPowershellEmpire.yaml",
+	"Solutions/CyberArkEPM/Analytic Rules/CyberArkEPMProcessChangedStartLocation.yaml",
+	"Solutions/CyberArkEPM/Analytic Rules/CyberArkEPMRareProcInternetAccess.yaml",
+	"Solutions/CyberArkEPM/Analytic Rules/CyberArkEPMRenamedWindowsBinary.yaml",
+	"Solutions/CyberArkEPM/Analytic Rules/CyberArkEPMUnexpectedExecutableExtension.yaml",
+	"Solutions/CyberArkEPM/Analytic Rules/CyberArkEPMUnexpectedExecutableLocation.yaml"
+],	
+"Workbooks": [
```

Comment:
@cyberark-bizdev, @CYBRJulie can you please the path for Workbook as it has not been added in solution. 

The correct path is 
Solutions/CyberArkEPM/Workbooks/CyberArkEPM.json